### ISSUE TITLE
Tweakaroo ghost blocks

### ISSUE THREAD

**Issue by TotallyThatSandwich** [maruohon/tweakeroo#345]
upon using tweakaroo Flexible and Accurate Block Placement the blocks are placed as ghost blocks, i can walk on them but if i try and place a block on top of them they disappear, i have a video of it here.

https://www.youtube.com/watch?v=LESYLT_k7Cs

**Comment by ThreePG** [comment]
I'm experiencing the same issues. The blocks also will become solid blocks if placing a solid block next to the original "ghost" block. Also, when destroying the ghost block as he shows in the video, you lose the item in your inventory until you update the item by selecting it in your hotbar. All my settings are the default settings when first installing the mod except the hotkey to activate the placement tweaks. Also, mine seems to do it when placing a block with the air gap in between, holding Left Control and selecting the center placement using Flexible placement. Perhaps in the video it's because he's placing them on transparent blocks as to why his doesn't need the air gap space for this to occur?

**Comment by maruohon** [comment]
Duplicate of #344. Also see the linked other issue from there for the longer explanation.

**Comment by KisenaMiyuki** [comment]
The problem is, despite saying that the issue was fixed for single player, we are still having the same issue in single player worlds with version 0.13.1.
Is there something we should tweak beforehand, or should we wait for another update?

**Comment by maruohon** [comment]
The last paragraph of the explanation here explains why it's still an issue, and why it still needs a "different fix": https://github.com/maruohon/litematica/issues/521

TL;DR; Only one mod can apply the fix, and currently it being active is tied to the related feature in the mod being enabled. So if you have both Litematica and Tweakeroo installed, I think the fix is only enabled if you have Easy Place enabled in  Litematica. Unless you use the previous version 0.11.1 of Litematica until the next version (0.11.3) comes out with the fix separated to a new config option that is not tied to Easy Place. And the same needs to happen for Tweakeroo as well.

**Comment by TotallyThatSandwich** [comment]
So is there any way to fix it?

**Comment by maruohon** [comment]
Like I said I need to release new versions of Tweakeroo and Litematica. I'll probably do it later today.

**Comment by maruohon** [comment]
I just uploaded Litematica 0.11.3 and Tweakeroo 0.13.3, which separate that fix/bypass to a new option that is not tied to any actual feature being enabled in the mods.
